src/app/constant.js

```diff
@@ -349,6 +349,13 @@ export const ZINDEXES = {
   }
 };
 
+/**
+ * @since v3.8
+ */
+export const API_BASE_URLS = {
+  CRS: '/crs/'  //Example /crs/<epsg_code>
+};
+
```

Comment:
before making it public, did you check that this variable wasn't already useless?

Reply:
@Raruto the idea is to provide a unique point where all api urls from server are listed